public/index.html

```diff
@@ -119,15 +119,15 @@
 
   <!-- Global site tag (gtag.js) - Google Ads: 527465415 -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=AW-527465415"></script>
-  <script>
+  <script async>
```

Comment:
Hrm...  this seems especially questionable since Google tells you to use `async` for the external script reference but not for this inline script.  Should we really be diverging from their example?

Reply:
That's a good point - I'm going to restore GA to what they recommend, but keep the async for the remaining scripts. FullStory is particularly heavy, and it's mostly used to record user behaviour.